ignite/handlers/param_scheduler.py

```diff
@@ -308,11 +320,13 @@ def __init__(
         )
         self.start_value = start_value
         self.end_value = end_value
-        self.cycle_size = int(cycle_size)  # Ensure cycle_size is integer
+        self.cycle_size = cycle_size
         self.cycle_mult = cycle_mult
         self.cycle = 0
         self.start_value_mult = start_value_mult
         self.end_value_mult = end_value_mult
+        self.warmup_duration = cyclic_warmup_duration
```

Comment:
Nit: why you have to names for this arg: warmup_duration and cyclic_warmup_duration ? Can we simply use warmup_duration everywhere?